imports/ui/pages/questions_library.jsx

```diff
@@ -12,13 +12,14 @@ import { QuestionEditItem } from '../QuestionEditItem'
 import { QuestionDisplay } from '../QuestionDisplay'
 import { QuestionSidebar } from '../QuestionSidebar'
 
-import { Questions, defaultQuestion } from '../../api/questions'
+import { Questions, defaultQuestion, questionQueries } from '../../api/questions'
```

Comment:
Remove the dependence on questionQueries.
The subscription will already apply the queries defined in the publications of api/questions.js.\
In this component, you only need to add things to the query that are based on the UI (so that you reduce the number of items sent to minimongo - no need to re-apply the stuff that is already checked in the publication).